packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -490,27 +351,24 @@ const methods = {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   parentKnowsIframeReady: (_unused: unknown) => {
     log("Method: `parentKnowsIframeReady` called");
-
+    // No UI change should happen in sight. Let the parent height adjust and in next cycle show it.
+    // Embed background must still remain transparent
     runAsap(function tryInformingLinkReady() {
-      if (!isLinkReady()) {
+      if (!isLinkReady({ embedStore })) {
         runAsap(tryInformingLinkReady);
         return;
       }
 
-      // No UI change should happen in sight. Let the parent height adjust and in next cycle show it.
-      // Embed background must still remain transparent
       makeBodyVisible();
-      if (isPrerendering()) {
-        log("prerenderState is 'completed'");
-        embedStore.prerenderState = "completed";
```

Comment:
`prerenderState` is now named `renderState` and we update it regardless of prerendering.

It helps conveying that if `connect` is possible